packages/esm-service-queues-app/src/index.ts

```diff
@@ -94,6 +95,10 @@ export const addProviderToRoomModal = getAsyncLifecycle(
 
 export const addQueueEntry = getSyncLifecycle(addQueueEntryComponent, options);
 
+export const queueTableName = getSyncLifecycle(QueueTableName, options);
+export const queueTablePriority = getSyncLifecycle(QueueTablePriority, options);
+export const queueTableStatus = getSyncLifecycle(QueueTableStatus, options);
```

Comment:
Use `getAsyncLifecycle()`. `getSyncLifecycle()` should basically just be used for links.